code/GridField/CopyButton.php

```diff
@@ -81,12 +81,13 @@ private function getCopyAction($gridField, $record, $columnName)
             'CopyRecord'.$record->ID,
             false,
             "copyrecord",
-            ['RecordID' => $record->ID])
-                                     ->addExtraClass('gridfield-button-copy')
-                                     ->setAttribute('classNames', 'font-icon-plus')
-                                     ->setAttribute('title', $title)
-                                     ->setDescription(_t('GridAction.COPY_DESCRIPTION', 'Copy'))
-                                     ->setAttribute('aria-label', $title);
+            ['RecordID' => $record->ID]
+        )
+        ->addExtraClass('gridfield-button-copy')
```

Comment:
Ah, sorry, I meant these calls to be one more tab in.

Can you make that change then squash your commits 😇 

thanks

Reply:
Like this?
Ill squash when approved.